### ISSUE TITLE
Authorization: Bearer null but with query string token

### ISSUE THREAD

**Issue by karlpatrickespiritu** [tymondesigns/jwt-auth#1014]
Requests with `token` query string param but `Authorization` header is `Bearer null` returns:

````
{
  "error": "token_not_provided"
}
````

I might be just missing something but shouldn't it just proccess the request although my `Authorization` header is null? Since I have the token as a query string param on the request.

**Comment by kohenkatz** [comment]
No, it treats the word `null` as a string `"null"`, which is not a valid token. By default, if you want to use a query parameter, you need to make sure you are not sending an `Authorization` header.

If you want to change this behavior, you can write your own version of `Tymon\JWTAuth\Http\Parser\AuthHeaders` which checks for the word `null` and ignores it. You will also need your own version of `Tymon\JWTAuth\Providers\LaravelServiceProvider` which overrides the `registerTokenParser()` function to use your custom header parser.
